### ISSUE TITLE
Cannot deploy on different context

### ISSUE THREAD

**Issue by gustavonalle** [mempool-space/mempool.space#17]
It seems all the instruction assume the deployment of the site will happen under ```/``` , is it possible to deploy it under ```/pool/``` for example, in case Nginx is already serving something under ```/```?

**Comment by wiz** [comment]
Yeah, we need this for various use cases such as btcpayserver-docker - adding it to the task list

**Comment by wiz** [comment]
Found an easy way using `--baseHref` option of `ng build` command to declare the base href at compile time
